applications/GeoMechanicsApplication/custom_elements/integration_scheme.h

```diff
@@ -0,0 +1,27 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Anne van de Graaf
+//
+
+#pragma once
+
+namespace Kratos
+{
+
+class IntegrationScheme
```

Comment:
I think the name is very clear, but might be confusing when thinking about the time integration schemes, which have very similar names.

Reply:
I have discussed this last week with @WPK4FEM and the bottom line was this name is probably the clearest one. Alternatives such as `NumericalIntegrationScheme` or `SpatialIntegrationScheme` are not better, or even unusual to use. So we decided to stick to this name. Feel free to suggest another one when it comes to mind.